litmus/PTX/Memalloy/RWC1.litmus

```diff
@@ -0,0 +1,20 @@
+PTX RWC1
+"Rf-c-g Mcta-d-Rg-Rg Fr-w-g Mgl-d-Wg-Rg Fr-c-g"
+"https://github.com/johnwickerson/memalloy/blob/master/ptx_testing/IRIW1.txt"
+"https://multicore.doc.ic.ac.uk/gpu-litmus/entries/gtx540-RWC+membar.cta+membar.gl-p0::p1:p2-xgyg.txt"
+{
+P0:r0=0;
+P1:r0=0;
+P1:r2=0;
+P2:r0=0;
+P2:r2=0;
+x=0;
+y=0;
+}
+ P0@cta 0,gpu 0           | P1@cta 1,gpu 0           | P2@cta 1,gpu 0           ;
+ ld r0, 1                 | ld.weak r0, x            | ld r0, 1                 ;
+ st.weak x, r0            | fence.sc.cta             | st.weak x, r0            ;
+                          | ld.weak r2, y            | fence.sc.sys             ;
```

Comment:
shouldn't it be `fence.sc.gpu`?

Reply:
Yeah all the `membar.gl` should be `fence.sc.gpu`, fixed now.